syntax = "proto3";

package auth;

service AuthService {
  rpc GetUser (GetUserRequest) returns (GetUserResponse);
}

message User {
  int64 id = 1;
  string username = 2;
}

message GetUserRequest {
  int64 user_id = 1;
}

message GetUserResponse {
  User user = 1;
}